### ISSUE TITLE
Corrupt Project

### ISSUE THREAD

**Issue by obviousjim** [maxillacult/ofxPostGlitch#1]
the xcode project file is corrupted =(

**Comment by soogbet** [comment]
same here. getting a message : 
ofxPostGlitch-master/example/ofxPostGlitchExample.xcodeproj cannot be opened because it is missing its project.pbxproj file.

[using Xcode 5.0.2]

**Comment by estevancarlos** [comment]
Confirmed.  It's missing that file.

**Comment by jnsdbr** [comment]
You can copy the project.pbxproj file from the empty ofx example project via right click -> show package contents and insert it the same way into the postglitch example. But it seems like there are a lot of files missing.

**Comment by technolife** [comment]
Same here... Any solutions ?

**Comment by ayanoco** [comment]
You can see how it goes by making the new project by yourself and copying the src and the other folders :)

**Comment by jmcordeiro** [comment]
yes, this last suggestion did the trick for me but I also had to change the word ofApp to testApp in the project.pbxproj file (for your reference, it appeared 12 times)

**Comment by robmint** [comment]
@jmcordeiro would you care to share the project file? I have used the xcode project file from an emptyExample but am still getting lots of build errors about NSObjCRuntime.h. I just want a glitch of my own....
